web/pandas/community/ecosystem.md

```diff
@@ -512,6 +512,21 @@ wasted).
 - ``vaex.from_pandas``
 - ``vaex.to_pandas_df``
 
+### [Hail Query](https://hail.is/)
+
+An out-of-core, preemptible-safe, distributed, dataframe library serving
+the genetics community. Hail Query ships with on-disk data formats,
+in-memory data formats, an expression compiler, a query planner, and a
+distributed sort algorithm all designed to accelerate queries on large
+matrices of genome sequencing data.
+
+It is often easiest to use Pandas to manipulate the summary statistics or
+other small aggregates produced by Hail. For this reason, Hail provides
+native import to and export from Pandas DataFrames:
+
+- [`Table.from_pandas`](https://hail.is/docs/0.2/hail.Table.html#hail.Table.from_pandas)
```

Comment:
Is there a link that doesn't hardcode these docs to 0.2 like "latest"?

Reply:
We do not currently have such a link, but I can add one. Give me a couple days!

Reply:
OK, updated!